### ISSUE TITLE
What HTML Editor do you use.

### ISSUE THREAD

**Issue by Appmantx** [Simbul/baker#902]
I am currently using Indesign CS6 witth In5 Extention. I want to know if there is a better way to make my magazines.
Thank You

**Comment by thejoepeach** [comment]
Hi there...

in5 is a nice extension, I use Hype but is a barehand experience, you need to do everything by hand, wich in5 do all in indesing and you can edit it further on Flux editor or another HTML editor... hope it can help

**Comment by divlv** [comment]
Win / Notepad++

**Comment by folletto** [comment]
I personally use Sublime Text, but it's pretty much all handcoded. :)

Did you try something like Adobe Edge Reflow?
http://html.adobe.com/edge/

**Comment by 153Noel** [comment]
I use Indesign to make a pdf. Then I use a program called epagecreator to generate the html files from the pdf. There are navigation features in epagecreator (epagecreator.net) that are really helpful for the end user. Because it is made for an ebook publisher, it outputs files in several different formats such as Kindle, Nook, etc. They have recently released a program to make a standalone app but I think it is a little pricey. I stumbled on Baker ebook Framework and I watched Peter Maxon's excellent videos and I think that I can make the app. It might take me awhile to figure it out. If anyone knows of a video other than Peter's I would appreciate a note. Thanks, Noel
